src/App.js

```diff
@@ -1,18 +1,31 @@
 import React from 'react';
+
+import { TodoList } from './components/TodoList/TodoList';
```

Comment:
```suggestion
import { TodoList } from './components/TodoList';
```
импортируй файл через index.js